@here/harp-mapview/lib/MapView.ts

```diff
@@ -2642,6 +2642,38 @@ export class MapView extends EventDispatcher {
         return this.m_raycaster;
     }
 
+    /**
+     * Returns the world space position from the given screen position.
+     *
+     * @remarks
+     * This method will always return a point. if the given `(x, y)` value is not intersecting
+     * the ground plane a fallback point on the far plane will be computed.
+     * Therfore the returned point might not be on the earth surface.
+     *
+     * @param x - The X position in css/client coordinates (without applied display ratio).
+     * @param y - The Y position in css/client coordinates (without applied display ratio).
+     */
+    getWorldPositionAtSafe(x: number, y: number): THREE.Vector3 {
+        this.m_raycaster.setFromCamera(this.getNormalizedScreenCoordinates(x, y), this.m_camera);
```

Comment:
2657 to 2662 seem to be a repeat of the `getWorldPositionAt` method. Why not just call that method and set its return value to `worldPos`?